hbase-http/src/main/java/org/apache/hadoop/hbase/http/ProfileServlet.java

```diff
@@ -355,10 +358,10 @@ private Output getOutput(final HttpServletRequest req) {
       try {
         return Output.valueOf(outputArg.trim().toUpperCase());
       } catch (IllegalArgumentException e) {
-        return Output.SVG;
+        return Output.HTML;
       }
     }
-    return Output.SVG;
+    return Output.HTML;
```

Comment:
1.8.5 does support the HTML output even thought the doc doesn't mention it. So this is compatible across v1 and v2.